lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -609,7 +609,11 @@ codecVersion = verMap ! nodeToClientVersion
     where verMap = supportedNodeToClientVersions (Proxy @(CardanoBlock StandardCrypto))
 
 codecConfig :: W.SlottingParameters -> CodecConfig (CardanoBlock c)
-codecConfig sp = CardanoCodecConfig (byronCodecConfig sp) ShelleyCodecConfig
+codecConfig sp = CardanoCodecConfig
+    (byronCodecConfig sp)
+    ShelleyCodecConfig
+    ShelleyCodecConfig
+    ShelleyCodecConfig
```

Comment:
Are those two additional other Codecs supposed to be for Mary and Allegra?

Reply:
I don't think so - not at the moment. There is this:
```
data instance BlockConfig (ShelleyBlock era) = ShelleyConfig {..}
```
It fits with AllegraEra and MaryEra.